index.js

```diff
@@ -66,6 +70,11 @@ yargs
         type: 'boolean',
         default: false
     })
+    .option('rust',{
+        desc: 'use rust for smart contract',
+        type: 'boolean',
+        default: true
```

Comment:
I think we shouldn't switch to Rust by default yet. Dev experience for Rust won't be smooth if you are just playing around with dev tools (too much stuff needed to pre-install vs just Node.js).

Reply:
done